scripts/release/mule/test/test.sh

```diff
@@ -21,17 +21,17 @@ PKG_DIR="./tmp/node_pkgs/$OS_TYPE/$ARCH_TYPE"
 
 pushd "$PKG_DIR"
 
-if ! $USE_CACHE
+if [ -n "$S3_SOURCE" ]
```

Comment:
Is this going to always download from s3 now, instead of being able to use already downloaded binaries? This might not mesh well with the folder structure the package pipeline uses, since the prefix isn't the same between os/arch's

Reply:
No, it will only download from s3 if `S3_SOURCE` is specified.